client/src/components/ItemCard/ItemCard.tsx

```diff
@@ -15,60 +21,65 @@ function ItemCard({
   const theme = useTheme();
   const themeSpacing = theme.spacing(1);
 
-  const increaseCounter = () => {
-    updateItemQuantity(cartItem.item.barcode, cartItem.quantity + 1);
-  };
-
-  const decreaseCounter = () => {
-    updateItemQuantity(cartItem.item.barcode, cartItem.quantity - 1);
+  const updateItemQuantityWrapper = (quantity: number) => {
+    updateItemQuantity(cartItem.item.barcode, quantity);
   };
 
   return (
-    <Card
+    <Paper
+      elevation={0}
       style={{
         marginTop: themeSpacing,
         padding: themeSpacing,
       }}
     >
-      <Grid container direction="row" justify="center" alignItems="center">
-        <Grid item xs={3}>
-          <p>Media</p>
-        </Grid>
-        <Grid item xs={3}>
-          <Typography variant="body1">{cartItem.item.name}</Typography>
+      <Grid container>
+        <Grid item>
+          <ItemCardMedia
+            height={ITEM_CARD_MAX_HEIGHT}
+            media={cartItem.item.media}
+          />
         </Grid>
-        <Grid item xs={2}>
-          <Typography variant="body1">
-            ${cartItem.item.price.toFixed(PRICE_FRACTION_DIGITS)}
-          </Typography>
-        </Grid>
-        <Grid item xs={2}>
-          <Typography variant="body1">${getSubtotalPrice(cartItem)}</Typography>
-        </Grid>
-        <Grid
-          container
-          item
-          xs={2}
-          spacing={0}
-          direction="column"
-          alignItems="center"
-        >
-          <Grid item xs={12}>
-            <button id="inc" onClick={increaseCounter}>
-              Plus
-            </button>
-          </Grid>
-          <Grid item xs={12}>
-            <p>{cartItem.quantity}</p>
+        <Grid item xs container direction="row">
+          <Grid item xs container direction="column" align-items="stretch">
+            <Grid item>
+              <Typography variant="body1">{cartItem.item.name}</Typography>
+            </Grid>
+            <Grid item xs style={{ overflow: "scroll" }}>
+              {cartItem.item.detail &&
+                parseRawTextNewlines(
+                  cartItem.item.detail
+                ).map((line: string) => [
+                  <Typography variant="subtitle2">{line}</Typography>,
+                  <Typography variant="subtitle2">{line}</Typography>,
+                ])}
```

Comment:
nit: I would probably make this a smaller react component for printing any multi-line text, maybe this is used in other places like Receipt.

Reply:
hmm, ok possible to explore, raised issue: #160